apm-collector/apm-collector-agentstream/src/main/java/org/skywalking/apm/collector/agentstream/worker/node/component/dao/NodeComponentEsDAO.java

```diff
@@ -18,12 +25,34 @@
         List<IndexRequestBuilder> indexRequestBuilders = new ArrayList<>();
         dataMap.forEach((id, data) -> {
             Map<String, Object> source = new HashMap();
-            source.put(NodeComponentTable.COLUMN_AGG, data.getDataString(1));
-            source.put(NodeComponentTable.COLUMN_TIME_BUCKET, data.getDataLong(0));
+            source.put(NodeComponentTable.COLUMN_APPLICATION_ID, data.getDataInteger(0));
```

Comment:
Elasticsearch source must Map or XContentBuilder and this map is a row data, like json object. why use list?